Assets/MixedRealityToolkit/_Core/Utilities/Scripts/DebugUtilities.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.InputSystem.Focus;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
+{
+    public static class DebugUtilities
+    {
+        public static void DebugAssert(bool condition, string message)
```

Comment:
This is where it would be cool to use (add?) System.Diagnostics.Debug.Assert()

Reply:
@SimonDarksideJ I thought we were waiting to add this?

Reply:
I removed the implementations, but left in the Shell utility for when I eventually wear you down :P